### DISCUSSION TITLE
ie/edge 浏览器中能打开，google 中打不开后台管理页面

### DISCUSSION THREAD

**Discussion by ghost** [rabbitmq/rabbitmq-server#3411] [Other]
[root@mid ~]# /sbin/service rabbitmq-server status
Redirecting to /bin/systemctl status rabbitmq-server.service
● rabbitmq-server.service - RabbitMQ broker
   Loaded: loaded (/usr/lib/systemd/system/rabbitmq-server.service; enabled; vendor preset: disabled)
   Active: active (running) since Fri 2021-09-10 18:16:00 CST; 2 days ago
 Main PID: 918 (beam)
   Status: "Initialized"
    Tasks: 68
   Memory: 126.5M
   CGroup: /system.slice/rabbitmq-server.service
           ├─  918 /usr/lib64/erlang/erts-5.10.4/bin/beam -W w -A 64 -P 1048576 -t 5000000 -stbt db -zdbbl 32000 -K true -- -root /usr/lib64/erlang -progname erl -- -home /var/lib/rabbitmq -- -pa /usr/lib/...
           ├─ 1112 /usr/lib64/erlang/erts-5.10.4/bin/epmd -daemon
           ├─ 1898 inet_gethost 4
           └─86100 inet_gethost 4

Sep 10 18:15:50 mid rabbitmq-server[918]: RabbitMQ 3.6.10. Copyright (C) 2007-2017 Pivotal Software, Inc.
Sep 10 18:15:50 mid rabbitmq-server[918]: ##  ##      Licensed under the MPL.  See http://www.rabbitmq.com/
Sep 10 18:15:50 mid rabbitmq-server[918]: ##  ##
Sep 10 18:15:50 mid rabbitmq-server[918]: ##########  Logs: /data/rabbitmq/log/rabbit@mid.log
Sep 10 18:15:50 mid rabbitmq-server[918]: ######  ##        /data/rabbitmq/log/rabbit@mid-sasl.log
Sep 10 18:15:50 mid rabbitmq-server[918]: ##########
Sep 10 18:15:50 mid rabbitmq-server[918]: Starting broker...
Sep 10 18:16:00 mid rabbitmq-server[918]: systemd unit for activation check: "rabbitmq-server.service"
Sep 10 18:16:00 mid systemd[1]: Started RabbitMQ broker.
Sep 10 18:16:01 mid rabbitmq-server[918]: completed with 6 plugins.

![image](https://user-images.githubusercontent.com/71524776/133060373-4181d8f5-7992-4507-b340-e2798880bb45.png)

![image](https://user-images.githubusercontent.com/71524776/133060432-9a4de13e-ab4d-48cc-8436-b10659bc6149.png)

![image](https://user-images.githubusercontent.com/71524776/133060503-aa3d2e36-27ea-4f4b-b06a-b1873ebfc306.png)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
RabbitMQ 3.6 has been [out of support for over three years](https://www.rabbitmq.com/versions.html). Consider [upgrading](https://www.rabbitmq.com/upgrade.html). One step that the upgrades guide recommends is clearing your browser cache.

You are also welcome to inspect JS console and Networking tabs in your browser's Developer Tools to see if there are any errors.
